packages/next-swc/crates/core/Cargo.toml

```diff
@@ -16,18 +16,18 @@ pathdiff = "0.2.0"
 serde = "1"
 serde_json = "1"
 styled_components = "0.17.0"
-swc = "0.138.0"
-swc_atoms = "0.2.7"
-swc_common = { version = "0.17.9", features = ["concurrent", "sourcemap"] }
-swc_css = "0.94.0"
-swc_ecma_loader = { version = "0.28.0", features = ["node", "lru"] }
-swc_ecmascript = { version = "0.123.0", features = ["codegen", "minifier", "optimization", "parser", "react", "transforms", "typescript", "utils", "visit"] }
+swc = "0.140.1"
```

Comment:
Is there a reason you used a non-recent version?

Reply:
It was to get the build to work. When I tried using a newer versions of these crates, the build failed (see the failed build here https://github.com/vercel/next.js/runs/5458285184?check_suite_focus=true#step:14:440)

I then started decreasing the versions of the crates until the build passed.